BootloaderCommonPkg/Library/ElfLib/Elf32.h

```diff
@@ -42,214 +35,218 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
  */
 
 #ifndef _SYS_ELF32_H_
-#define _SYS_ELF32_H_   1
+#define _SYS_ELF32_H_ 1
+
 
-#include "ElfCommon.h"
+/*
+ * ELF definitions common to all 32-bit architectures.
+ */
 
-//
-// ELF definitions common to all 32-bit architectures.
-//
-typedef UINT32       ELF32_ADDR;
-typedef UINT16       ELF32_HALF;
-typedef UINT32       ELF32_OFF;
-typedef INT32        ELF32_SWORD;
-typedef UINT32       ELF32_WORD;
-typedef UINT64       Elf32_LWORD;
+typedef UINT32  Elf32_Addr;
+typedef UINT16  Elf32_Half;
+typedef UINT32  Elf32_Off;
+typedef INT32    Elf32_Sword;
+typedef UINT32  Elf32_Word;
+typedef UINT64  Elf32_Lword;
```

Comment:
Is the newly added ones from EDKII? 
Looks the removed one following EDKII coding style but the newly added one not.